src/panoptoindexconnector/implementations/graph.yaml

```diff
@@ -0,0 +1,58 @@
+#
+# Panopto index connector configuration file
+#
+
+
+# The address to your panopto site
+panopto_site_address: https://your.site.panopto.com
+
+# The oauth credentials to connect to the panopto API
+panopto_oauth_credentials:
+    username: myconnectoruser
+    password: mypassword
+    client_id: 123
+    client_secret: 456
+    grant_type: password
+
+# Your index integration target endpoint
+target_address: https://graph.microsoft.com/v1.0/external/connections
+
+# Your graph data for the connector
+target_credentials:
+    tenant_id: 00000000-0000-0000-0000-000000000000
+    client_id: 00000000-0000-0000-0000-000000000000
+    client_secret: myclientsecret
+    grant_type: client_credentials
+    scopes: ["https://graph.microsoft.com/.default"]
+    authority: https://login.microsoftonline.com
+
+# Your graph connection (connector) that will be used to push Panopto items to
+target_connection:
+    # 'id' can only have ASCII alphanumeric characters and no empty spaces.
+    id: sampleConnectionId
+    name: Sample connection name
+    description: Sample connection description
+
+# The name of your implementation
+target_implementation: graph_implementation
+
```

Comment:
Please make sure you add this line:

```
# Set to "true" if we should not push permissions to the target;
# often used with the principal_allowlist to control permissions by
# what is synced rather than matching the ID Provider on the target
skip_permissions: false
```

Reply:
Done